gulp-tasks/build.js

```diff
@@ -169,6 +170,7 @@ gulp.task('build:showcase', function() {
   });
 
   // Generate the RSS & ATOM feeds
+  options.title = 'Show Cases';
```

Comment:
one word?

Reply:
All other instances in here are two words.